source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,258 @@
+      collections to the list in the ``"collections"`` field for that
+      database entry.
+
+   .. step:: Use the Filter.
+
+      To use the filter, attach the filter json when you make the
+      :ref:`/start <c2c-api-start>` API call to begin syncing.
+
+      .. literalinclude:: /includes/api/requests/start-filtered.sh
+         :language: shell
+
+For an example configuration, see: :ref:`c2c-example-start-with-filter`.
+
+.. _c2c-change-filter:
+
+Replace an Existing Filter
+--------------------------
+
+You cannot update an existing filter. You must stop the ongoing sync
+process, prepare the destination cluster, and restart ``mongosync`` with
+a new filter. 
+
+When ``mongosync`` ran your original filter, it created databases with
+your data ("user databases") and the
+``mongosync_reserved_for_internal_use`` system database on the
+destination cluster. You must remove those databases before restarting
+``mongosync`` with your new filter.
+
+Follow these steps to prepare the destination cluster for a new filter.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Remove ``mongosync_reserved_for_internal_use``.
+
+      a. Stop the ``mongosync`` process. 
+      
+      #. Connect to the destination cluster with :binary:`mongosh`. If
+         the destination is a sharded cluster, connect to the
+         :binary:`mongos` instance. If the destination is a replica set,
+         connect to the primary :binary:`mongod` instance.
+
+      #. Drop the ``mongosync_reserved_for_internal_use`` system
+         database.
+
+         .. code-block:: javascript
+          
+            use mongosync_reserved_for_internal_use
+            db.dropDatabase()
+
+   .. step:: Remove user databases.
+
+      a. List the databases in the cluster
+
+         .. code-block:: javascript
+          
+            show databases
+
+      #. Remove user databases. The ``admin``, ``local``, and ``config``
+         databases are system databases. You should not edit these
+         system databases without instructions from MongoDB support. 
+         
+         If the ``show databases`` command lists any user databases on
+         the destination cluster, you must remove them.
+
+         Repeat this step for each user database list:
+
+         .. code-block:: javascript
+          
+            use <user database name>
+            db.dropDatabase()
+
+         Note: After the first :method:`db.dropDatabase()` operation
+         completes, you may need to run it a second time to remove the
+         database.
+
+   .. step:: Configure a New Filter.
+
+      a. Decide which databases and collections you want to filter on.
+         Add the databases and collections to the ``includeNamespaces``
+         array. For configuration details, see
+         :ref:`c2c-configure-filter`.
+
+      #. Run ``mongosync`` to :ref:`reconnect <c2c-conn-top-level>` the
+         source and destination clusters.
```

Comment:
```suggestion
      #. Run ``mongosync`` to :ref:`reconnect <c2c-conn-top-level>` to the
         source and destination clusters.
```

Reply:
Done - I think this works either way, adding 'to' for clarity